src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -166,6 +166,11 @@ private static JObject ToJson(StackItem item, Session session)
 
         private static Signer[] SignersFromJson(JArray _params, ProtocolSettings settings)
         {
+            if (_params.Count > Transaction.MaxTransactionAttributes)
+            {
+                throw new RpcException(-100, "Max allowed witness exceeded.");
```

Comment:
> We can wait for https://github.com/neo-project/neo-modules/pull/815

Agree, some InvalidParams error can be used here. I'll review #815 one more time ASAP.